pkgs/development/compilers/ante/default.nix

```diff
@@ -0,0 +1,51 @@
+{ fetchFromGitHub
+, lib
+, libffi
+, libxml2
+, llvmPackages_13
+, ncurses
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "ante";
+  version = "unstable-2022-08-21";
+  src = fetchFromGitHub {
+    owner = "jfecher";
+    repo = "ante";
+    rev = "832a8545fab6cdfeb0215bbdd774e48679a7611b";
+    sha256 = "sha256-AGsAsdnjPFQUaeS4eai0gpyfJ8wLvND6YKvEc427a7s=";
+  };
+  cargoSha256 = "sha256-WwwV7UCFMcUt+IbF2Nvz0MFlCbayiCfneSHauPo3Rlw=";
+  cargoDepsName = pname;
```

Comment:
Can you please explain with a comment why this is required? I could build it without this (and with a different cargoSha256).

Reply:
It's not actually required. I only set the `cargoDepsName` since it was mentioned in the rust framework docs and didn't think stop to think if it had to be there. There's actually no reason to override the name vendored dependencies. I'm gonna fixup that change.